platform/app/page.tsx

```diff
@@ -143,7 +143,7 @@ export default async function Home() {
                 <td style={{ textAlign: "center" }}>4</td>
               </tr>
               <tr>
-                <td>The team took on an ambitious challenge, through the scale &amp; scope of the problem, their tech stack, or otherwise impressive medium.</td>
+                <td>The team took on anambitious challenge, through the scale &amp; scope of the problem, their tech stack, or otherwise impressive medium.</td>
```

Comment:
```suggestion
                <td>The team took on an ambitious challenge, through the scale &amp; scope of the problem, their tech stack, or otherwise impressive medium.</td>
```